crates/vision/src/field_border_detection.rs

```diff
@@ -112,13 +112,21 @@ fn find_border_lines(
     second_line_association_distance: f32,
 ) -> Vec<Line2> {
     // first line
-    let result = ransac.next_line(20, first_line_association_distance);
+    let result = ransac.next_line(
+        20,
```

Comment:
Not your changes, but maybe we can make this a parameter? :)

Reply:
Done